utils/slugifyToIRI.js

```diff
@@ -1,5 +1,5 @@
 export default function slugifyToIRI(input) {
-  return `http://regen.network/${input
+  return `regen:${input
```

Comment:
so we can just use the prefix here because the context is included wherever this function is being used?

Reply:
this is more to simplify the IRI a little bit, assuming we know what `regen:` means since it's a project of ours
similarly to the `regen:` prefix we use in on-chain metadata IRIs